index.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Quiz-Miniac</title>
+</head>
+<body>
+    <div class="container">
+        
```

Comment:
as you used semantic tags section below, you can use main tag here instead of div

Reply:
Since these changes are in Index.html i can only change them once merged and i'll do them. Thank You soo much